src/test/java/model/wrapper/PositionTest.java

```diff
@@ -22,15 +22,15 @@ void beforeEach() {
     @Test
     @DisplayName("4 이상의 값이 생성되면 한칸 전진한다.")
     void move() {
-        position.move(new StubMovableRacingNumberGenerator());
+        position.move(new StubMovableRacingNumberGenerator().generate());
```

Comment:
```suggestion
        position.move(5);
```

여기서는 Generator와 상관없이 테스트를 짜는 것은 어떨까요?

Reply:
말씀하신 바가 맞습니다...

단순히 값만 건내주는데 굳이 Generator를 사용할 필요가 없네요...
감사합니다!